addons/web_editor/static/src/js/wysiwyg/wysiwyg_iframe.js

```diff
@@ -92,7 +92,7 @@ Wysiwyg.include({
      */
     _loadIframe: function () {
         var self = this;
-        this.$iframe = $('<iframe class="wysiwyg_iframe">').css({
+        this.$iframe = $('<iframe class="wysiwyg_iframe o_iframe">').css({
```

Comment:
Commit message: mention we merge this in 15.3 as actually needed to remove the race condition on a test introduced by https://github.com/odoo/odoo/commit/52d32e26c34bef10823582ef5c1f840492b73d7a and that it could be backported if needed.

Also: was it actually tested that it removed the race condition @bvr-odoo ?

Reply:
Why do you need to add a new class? Is there a problem using `wysiwyg_iframe` ?

Reply:
- Not putting wysiwyg_iframe in web_tour
- Having a generic class for all iframes where we want this behavior